apps/nextjs-website/src/components/organisms/Auth/LoginForm.tsx

```diff
@@ -1,19 +1,149 @@
+        },
+      }}
+    >
+      <Card variant='elevation' elevation={8} sx={{ borderRadius: '16px' }}>
+        <Grid container justifyContent='center'>
+          <Grid item xs={10}>
+            <form>
+              <Stack pt={4} display='flex' alignItems='center'>
+                <IllusLogin />
+              </Stack>
+              <Typography variant='h3' pt={8} mb={4} textAlign='center'>
+                {login.loginToYourAccount}
+              </Typography>
+              <Stack spacing={2} mb={4}>
+                <TextField
+                  label={shared.emailAddress}
+                  variant='outlined'
+                  size='small'
+                  onChange={(e) => setUsername(e.target.value)}
+                  sx={{
+                    backgroundColor: 'white',
+                  }}
+                />
+              </Stack>
+              <Stack spacing={2} mb={2}>
+                <FormControl variant='outlined'>
+                  <InputLabel htmlFor='password-input' sx={{ top: '-8px' }}>
+                    {shared.password}
+                  </InputLabel>
+                  <OutlinedInput
+                    id='password-input'
+                    type={showPassword ? 'text' : 'password'}
+                    onChange={(e) => setPassword(e.target.value)}
+                    endAdornment={
+                      <InputAdornment position='end'>
+                        <IconButton
+                          aria-label='toggle password visibility'
+                          onClick={handleClickShowPassword}
+                          onMouseDown={handleMouseDownPassword}
+                          edge='end'
+                        >
+                          {showPassword ? <VisibilityOff /> : <Visibility />}
+                        </IconButton>
+                      </InputAdornment>
+                    }
+                    label={shared.password}
+                    inputProps={{
+                      sx: {
+                        padding: '8.5px 14px',
+                      },
+                    }}
+                  />
+                </FormControl>
+              </Stack>
+              <Grid container mb={1}>
+                <FormControlLabel
+                  control={<Checkbox />}
+                  label={login.rememberMe}
+                />
+              </Grid>
+              <Stack spacing={4} pt={4} pb={8}>
+                <Stack direction='row' justifyContent='center'>
+                  <Button variant='contained' onClick={onLogin}>
+                    {shared.login}
+                  </Button>
+                </Stack>
+              </Stack>
+              <Divider />
+              <Stack spacing={4} pt={4} pb={8}>
+                <Typography variant='body2' textAlign='center'>
+                  {login.noAccount}{' '}
+                  <Link href='/auth/sign-up'>
+                    <Button variant='text'>{shared.signUp}</Button>
+                  </Link>
```

Comment:
Why is this `Link` within the `Typography`?

or, if it is required to the `Link` to be within the `Typography`, is the `Stack` necessary?

Reply:
Changed accordingly to Figma